main.nf

```diff
@@ -121,6 +121,7 @@ def helpMessage() {
       --protein_level_fdr_cutoff    Protein level FDR cutoff (this affects and chooses the peptides used for quantification)
 
       Quantification:
+      --quantification_method       Quantification method supported by proteomicslfq ('feature_intensity' or 'spectral_counting', default: 'feature_intensity')
```

Comment:
This also needs to be added to the nextflow.config with a default value:
`params.quantification_method = "feature_intensity"` and in the nextflow_schema.json under the Quantification group with a help text and the allowed strings.
And you need to actually use it in the "proteomicslfq" process with the "params.quantification".
If you want you could add this with a test config for CI as well.

Reply:
- [x] is already added in the proteomicslfq (https://github.com/nf-core/proteomicslfq/blob/30cf776a7e622a962ae5fc2860a3853d8518acbd/main.nf#L1070). 
- [x] It is already in the nextflow.config (https://github.com/nf-core/proteomicslfq/blob/30cf776a7e622a962ae5fc2860a3853d8518acbd/nextflow.config#L93) 
- [ ] just added to the json (https://github.com/nf-core/proteomicslfq/pull/91/commits/fd3185cb28e855d4922f800ebec79f55890911db#diff-2d9d6ae344b889545d73f7c118b3f05aR695)